front/src/modules/timetableItem/components/Timetable/FilterPanel.tsx

```diff
@@ -63,13 +59,13 @@ const FilterPanel = ({ toggleFilterPanel, timetableFilters }: FilterPanelProps)
   ];
 
   const formattedCategoryOptions: {
-    id: 'all' | 'noCategory' | TrainMainCategory;
+    id: 'all' | 'noCategory' | (TrainMainCategories | SubCategory['code']);
```

Comment:
You don't need the `()` since it's an `or`
```suggestion
    id: 'all' | 'noCategory' | TrainMainCategories | SubCategory['code'];
```

Reply:
I preferred to leave it because I found it easier to understand. To clearly understand that this is a category or subcategory.